src/sig_stfl/sig_stfl.c

```diff
@@ -0,0 +1,1435 @@
+	} else if (0 == strcasecmp(method_name, OQS_SIG_STFL_alg_lms_sha256_h10_w4_h5_w8)) {
+#ifdef OQS_ENABLE_SIG_STFL_lms_sha256_h10_w4_h5_w8
+		return OQS_SIG_STFL_alg_lms_sha256_h10_w4_h5_w8_new();
+#else
+		return NULL;
+#endif
+	} else if (0 == strcasecmp(method_name, OQS_SIG_STFL_alg_lms_sha256_h10_w8_h5_w8)) {
+#ifdef OQS_ENABLE_SIG_STFL_lms_sha256_h10_w8_h5_w8
+		return OQS_SIG_STFL_alg_lms_sha256_h10_w8_h5_w8_new();
+#else
+		return NULL;
+#endif
+	} else if (0 == strcasecmp(method_name, OQS_SIG_STFL_alg_lms_sha256_h10_w2_h10_w2)) {
+#ifdef OQS_ENABLE_SIG_STFL_lms_sha256_h10_w2_h10_w2
+		return OQS_SIG_STFL_alg_lms_sha256_h10_w2_h10_w2_new();
+#else
+		return NULL;
+#endif
+	} else if (0 == strcasecmp(method_name, OQS_SIG_STFL_alg_lms_sha256_h10_w4_h10_w4)) {
+#ifdef OQS_ENABLE_SIG_STFL_lms_sha256_h10_w4_h10_w4
+		return OQS_SIG_STFL_alg_lms_sha256_h10_w4_h10_w4_new();
+#else
+		return NULL;
+#endif
+	} else if (0 == strcasecmp(method_name, OQS_SIG_STFL_alg_lms_sha256_h10_w8_h10_w8)) {
+#ifdef OQS_ENABLE_SIG_STFL_lms_sha256_h10_w8_h10_w8
+		return OQS_SIG_STFL_alg_lms_sha256_h10_w8_h10_w8_new();
+#else
+		return NULL;
+#endif
+	} else if (0 == strcasecmp(method_name, OQS_SIG_STFL_alg_lms_sha256_h15_w8_h5_w8)) {
+#ifdef OQS_ENABLE_SIG_STFL_lms_sha256_h15_w8_h5_w8
+		return OQS_SIG_STFL_alg_lms_sha256_h15_w8_h5_w8_new();
+#else
+		return NULL;
+#endif
+	} else if (0 == strcasecmp(method_name, OQS_SIG_STFL_alg_lms_sha256_h15_w8_h10_w8)) {
+#ifdef OQS_ENABLE_SIG_STFL_lms_sha256_h15_w8_h10_w8
+		return OQS_SIG_STFL_alg_lms_sha256_h15_w8_h10_w8_new();
+#else
+		return NULL;
+#endif
+	} else if (0 == strcasecmp(method_name, OQS_SIG_STFL_alg_lms_sha256_h15_w8_h15_w8)) {
+#ifdef OQS_ENABLE_SIG_STFL_lms_sha256_h15_w8_h15_w8
+		return OQS_SIG_STFL_alg_lms_sha256_h15_w8_h15_w8_new();
+#else
+		return NULL;
+#endif
+	} else if (0 == strcasecmp(method_name, OQS_SIG_STFL_alg_lms_sha256_h20_w8_h5_w8)) {
+#ifdef OQS_ENABLE_SIG_STFL_lms_sha256_h20_w8_h5_w8
+		return OQS_SIG_STFL_alg_lms_sha256_h20_w8_h5_w8_new();
+#else
+		return NULL;
+#endif
+	} else if (0 == strcasecmp(method_name, OQS_SIG_STFL_alg_lms_sha256_h20_w8_h10_w8)) {
+#ifdef OQS_ENABLE_SIG_STFL_lms_sha256_h20_w8_h10_w8
+		return OQS_SIG_STFL_alg_lms_sha256_h20_w8_h10_w8_new();
+#else
+		return NULL;
+#endif
+	} else if (0 == strcasecmp(method_name, OQS_SIG_STFL_alg_lms_sha256_h20_w8_h15_w8)) {
+#ifdef OQS_ENABLE_SIG_STFL_lms_sha256_h20_w8_h15_w8
+		return OQS_SIG_STFL_alg_lms_sha256_h20_w8_h15_w8_new();
+#else
+		return NULL;
+#endif
+	} else if (0 == strcasecmp(method_name, OQS_SIG_STFL_alg_lms_sha256_h20_w8_h20_w8)) {
+#ifdef OQS_ENABLE_SIG_STFL_lms_sha256_h20_w8_h20_w8
+		return OQS_SIG_STFL_alg_lms_sha256_h20_w8_h20_w8_new();
+#else
+		return NULL;
+#endif
+	}
+//#endif //OQS_ENABLE_SIG_STFL_LMS
+	else {
+		return NULL;
+	}
+}
+
+OQS_API OQS_STATUS OQS_SIG_STFL_keypair(const OQS_SIG_STFL *sig, uint8_t *public_key, OQS_SIG_STFL_SECRET_KEY *secret_key) {
+#ifndef OQS_ALLOW_SFTL_KEY_AND_SIG_GEN
```

Comment:
```suggestion
#ifndef OQS_ALLOW_STFL_KEY_AND_SIG_GEN
```

Reply:
done in https://github.com/open-quantum-safe/liboqs/pull/1650/commits/fbb09d1b4eafc14fe231dbe8da8c4d60b9bdcd79